tensorflow/compiler/xla/service/gpu/conv_layout_normalization_test.cc

```diff
@@ -109,10 +109,11 @@ ENTRY TestComputation {
 )";
 
   RunAndCompareWithLayoutsNormalized(hlo);
-
+#if !TENSORFLOW_USE_ROCM
   MatchOptimizedHlo(hlo, R"(
 // CHECK: (f32[8,32,4,5,5]{4,3,2,1,0}, u8[0]{0}) custom-call([[bitcast_8_0:%[^ ]+]], [[fusion_1:%[^ ]+]], [[bias_2:%[^ ]+]]), window={size=3x3x3 pad=1_1x1_1x1_1}, dim_labels=bf012_oi012->bf012, custom_call_target="__cudnn$convBiasActivationForward"
   )");
```

Comment:
Is there an else case here for what appropriate on ROCM we can check?

Reply:
Not really. In case of CUDA, it verifies that the convolution is fused. On ROCM, I disable fusing for 3D convolutions (seems to be a MIOpen bug).